packages/webapp/src/containers/SensorReadings/styles.module.scss

```diff
@@ -16,3 +16,24 @@
     padding: 24px;
   }
 }
+
+.forecastInfo {
+  border: solid 1px var(--grey400);
+  color: var(--grey900);
+  border-radius: 4px;
+  padding: 24px 20px;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 18px;
+}
```

Comment:
This is entirely my fault because I messed around with the padding for almost every element in this view (sorry!) but I think a `margin-inline: 8px` would be needed to get the borders of the forecast box to line up with the nav bar above and the title below as in the Jira ticket.

Alternatively, I'm not opposed to removing the 8px inline padding for the nav component if you would like to take up full width again! But I think they should probably line up.

![Screenshot 2023-05-24 at 11 46 12 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/2a9b19ce-2fcc-4f1b-8ec6-a0828b2b6c2a)